### PR TITLE
Global header - dynamic navigation

### PR DESCRIPTION
**Pull request by ian-leggett**
## Problem
The navigation list items should show only show links to apps that you have permission to use.

screenshot 2019-02-22 at 09 43 44

## Solution
Only show links to apps that you have permission to use by reading your permitted apps list via SSO on the user response object.

### CODE REVIEW

File: src/data/schema/backend/user/me.schema.json [hunk 1]
```diff
@@ -1,24 +1,49 @@
 {
 	"$schema": "http://json-schema.org/draft-04/schema#",
 	"title": "User info",
-
 	"type": "object",
-	"required": [ "name", "email", "last_login" ],
+	"required": [
+		"name",
+		"email",
+		"last_login"
```
Comment (web-bert) [inline src/data/schema/backend/user/me.schema.json]: @ian-leggett this needs `permitted_applications` added to it

File: src/app/controllers/controller.index.js [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
+		const globalNavItems = res.locals.globalNavItems;
 		const sectorTeams = data.sectorTeams.results;
 		const overseasRegionGroups = data.overseasRegionGroups.results;
 		const ukRegions = data.ukRegions.results;
 		const globalHvcs = data.globalHvcs.results;
-		const summary = globalSummary.create( data.globalWins );
+		const summary = globalSummary.create(data.globalWins);
 
-		res.render( 'index.html', { sectorTeams, overseasRegionGroups, ukRegions, globalHvcs, summary } );
+		res.render('index.html', { globalNavItems, sectorTeams, overseasRegionGroups, ukRegions, globalHvcs, summary });
```
Comment (web-bert) [inline src/app/controllers/controller.index.js]: I don't think `globalNavItems` is need here as it's already on `res.locals` ?

Comment (ian-leggett) [reply src/app/controllers/controller.index.js]: Just removed this and app seems to be fine

File: src/data/schema/backend/user/me.schema.json (hunk 1 above)
Comment (ian-leggett) [reply src/data/schema/backend/user/me.schema.json]: Added 👍

**Pull request merged**